### PR TITLE
[138] Product Page using HTML

### PR DESCRIPTION
**Pull request by Blessing-Hubb**
##task : Create a product page using html
I added a product page with screenshots
Screenshot1
Screenshot2

### CODE REVIEW

File: submissions/blessingOlusuji/HTML/product.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ <!---This session is about product name and image-->
+ <session>
+      <div>
+          <img src="images/mobile1.jpg" width="300px" height ="300px">
+          <h4><u>Nokia Edge</u> : <strong>#110,000</strong></h4>
+          <p> Nokia Edge smartphone runs on Android v7.0 (Nougat) operating system. <br>
+              The phone is powered by octa core (2 GHz, quad core, cortex A53 + 1.2 GHz,<br>
+              quad core, cortex A53) processor. It runs on the mediaTek MT6755 chipset. <br>
+              It has 3GB RAM and 32 GB internal storage.
+          </P>
```
Comment (Ireti203) [inline submissions/blessingOlusuji/HTML/product.html:38]: Kindly use the small lettered p in the closing tag for the paragraph.

File: submissions/blessingOlusuji/HTML/product.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   <div>
+        <ul>
+            <li><a href ="#"> Reviews</a></li> 
+            <li><a href ="#"> FAQs</a></li> 
+            <li><a href ="#"> Blogs</a></li> 
+       </ul>
+       <p>&copy:Spotless-B 2020;</p>
+  </div>
+</footer>
+
```
Comment (Ireti203) [inline submissions/blessingOlusuji/HTML/product.html:70]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/blessingOlusuji/HTML/product.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   <h2><i>Spotless-B Mobile</i></h2>
+   <nav>
+      <ul>
+        <li><a href ="#">Home</a></li>
+        <li><a href ="#">Cart</a></li>
+        <li><a href ="#">About us</a></li>
+        <li><a href ="#">Contact us</a></li>
+      </ul>
+   </nav>
+
```
[the remark this answers is not in the document]
Comment (Blessing-Hubb) [reply submissions/blessingOlusuji/HTML/product.html:17]: Please what do you mean by whitespace here?

File: submissions/blessingOlusuji/HTML/product.html (hunk 2 above)
Comment (Blessing-Hubb) [reply submissions/blessingOlusuji/HTML/product.html:70]: Please, what do you mean by white space here?

Comment (Ireti203) [reply submissions/blessingOlusuji/HTML/product.html:70]: The empty line..

Comment (Blessing-Hubb) [reply submissions/blessingOlusuji/HTML/product.html:70]: Okay, I just checked but there isnt any, however, I have closed up the lines between closing body and html tag. I will be pushing modified version now, thanks.

**Comment by Blessing-Hubb** [comment]
Done

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**